app/controllers/concerns/idv/acuant_concern.rb

```diff
@@ -2,18 +2,12 @@
 
 module Idv
   module AcuantConcern
-    def acuant_sdk_ab_test_analytics_args
-      return {} if document_capture_session_uuid.blank?
-
-      {
-        acuant_sdk_upgrade_ab_test_bucket:
-          AbTests::ACUANT_SDK.bucket(document_capture_session_uuid),
-      }
-    end
```

Comment:
👋🏿 @matthinz ... looks like acuant_sdk_upgrade_ab_test_bucket will no longer be logged.  not opposed, just curious as to you reason why?

Reply:
Yep, bucket will now be present on `ab_tests.acuant_sdk.bucket`